.github/workflows/test-e2e.yml

```diff
@@ -0,0 +1,57 @@
+# Compile and run E2E tests using Cypress.
+name: Test E2E
+on:
+  # Run manually for now.
+  workflow_dispatch:
+  # NB: Dicuss whether this workflow should be configured to run once
```

Comment:
Nice to have the option to triggering as a workflow - I expect likely we will later integrate with the labelling system to define when/if to run but for now this is more than sufficient